lib/mapping/index.d.ts

```diff
@@ -128,27 +128,27 @@ export namespace mapping {
     update(doc: any, docInfo?: UpdateDocInfo): ModelBatchItem;
   }
 
-  interface ModelMapper {
+  interface ModelMapper<T = { [key: string]: any }> {
     name: string;
     batching: ModelBatchMapper;
 
-    get(doc: any, docInfo?: { fields?: string[] }, executionOptions?: string | MappingExecutionOptions): Promise<any>;
+    get(doc: { [key: string]: any }, docInfo?: { fields?: string[] }, executionOptions?: string | MappingExecutionOptions): Promise<null | T>;
```

Comment:
I've noticed that this you are using the `null` type here. If we use it here, we should use it in `Result<T>#first()` method.

I personally can't wrap my head about `Type | null` in ts definitions. With strict null checking, this succeeds:

```typescript
function get(): Video {
  // ...
}

const o: Video = get();
// this condition compiles (!)
if (o === null) {

}
```

Reply:
I dont see where you have enabled strict null checking right now.
IIRC there is no tsconfig.json

Reply:
> I've noticed that this you are using the `null` type here. If we use it here, we should use it in `Result<T>#first()` method.
> 
Yes. I will change that :)

Reply:
> I dont see where you have enabled strict null checking right now.

I mean as a general concept (not related to the patch), it feels weird for me to allow `o === null` when strictNullCheck is on (and `o` is not nullable)